### PR TITLE
Fix link to hanselman asp.net core article

### PR DESCRIPTION
**Pull request by mash1t**

### CODE REVIEW

**Comment by mash1t** [comment]
There still is an error concerning link testing:

`checking 605 GitHub repositories...
❌ ERROR {
  "0": {
    "show": true,
    "duplicates": "",
    "other_links_error": "",
    "github_repos": [
      {
        "message": "Unexpected end of document"
      }
    ]
  }
}`

I'll have a look at this this afternoon, open to suggestions on what could cause the issue.

**Comment by veggiemonk** [comment]
Thanks, I’ll look at why it breaks

**Comment by agebhar1** [comment]
still occurs after re-run

File: README.md [hunk 1]
```diff
@@ -324,7 +324,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker)
 - [portworx](https://portworx.com) :heavy_dollar_sign: - Decentralized storage solution for persistent, shared and replicated volumes.
 - [quobyte](https://www.quobyte.com/) :heavy_dollar_sign: - fully fault-tolerant distributed file system with a docker volume driver
 - [REX-Ray](https://github.com/rexray/rexray) provides a vendor agnostic storage orchestration engine. The primary design goal is to provide persistent storage for Docker, Kubernetes, and Mesos. By[@thecodeteam](https://github.com/thecodeteam) (DELL Technologies)
-- [Storidge](https://github.com/Storidge/quick-start#step-1-install-cio-software) :heavy_dollar_sign: - Software-defined Persistent Storage for Kubernetes and Docker Swarm
```
Comment (mash1t) [inline README.md:327]: ok, it's been an issue with an anchor link reference which currently the script pull_request.js has problems coping with.

**Review by mash1t — COMMENTED** [review]
ok, it's been an issue with an anchor link reference which currently the script pull_request.js has problems coping with.

**Review by veggiemonk — APPROVED** [review]
Let's merge this for the time being

**Pull request merged**